src/searches/useProductTypeSearch.ts

```diff
@@ -26,6 +27,26 @@ export const searchProductTypes = gql`
   }
 `;
 
+export function useSearchProductTypes() {
+  const client = useApolloClient();
+
+  return (query: string) =>
+    client
+      .query<SearchProductTypesQuery, SearchProductTypesQueryVariables>({
+        query: SearchProductTypesDocument,
+        variables: {
+          first: 10,
+          query,
+        },
+      })
+      .then(({ data }) =>
+        mapEdgesToItems(data.search).map(({ name, id }) => ({
+          label: name,
+          value: id,
+        })),
+      );
+}
+
```

Comment:
**question**: Why is it implemented? Can't we use `useProductTypeSearch` instead?